Web3Swift/StringScalar/TrimmedPrefixString.swift

```diff
@@ -38,15 +40,12 @@ public final class TrimmedPrefixString: StringScalar {
     public func value() throws -> String {
         let string = try self.string.value()
         let prefix = try self.prefix.value()
-        if string.hasPrefix(prefix) {
-            return String(
-                string.dropFirst(
-                    Int(prefix.count)
-                )
-            )
-        } else {
-            return string
+        guard let range = string.range(of: "^(\(prefix)){1,}", options: [.regularExpression]) else { return string }
```

Comment:
Please put `of:`, `options:` and conditionals on separate lines. See `HexString.value()` for reference